Services/CO.CDP.DataSharing.WebApi.Tests/CO.CDP.DataSharing.WebApi.Tests.csproj

```diff
@@ -11,7 +11,9 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="FluentAssertions" Version="6.12.0" />
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+        <PackageReference Include="Nanoid" Version="3.1.0" />
```

Comment:
This is already in CO.CDP.DataSharing.WebApi.csproj, so shouldn't be needed here.

Reply:
done.